src/common.c

```diff
@@ -1100,6 +1100,18 @@ int processors = 1;
 long get_system_cpus(void) {
     processors = 1;
 
+    #ifdef __APPLE__
+        int32_t tmp_processors;
+
+        if (unlikely(GETSYSCTL("hw.logicalcpu", tmp_processors))) {
```

Comment:
From man 3 sysctl:

> hw.ncpu
             The number of cpus. This attribute is deprecated and it is recom-
             mended that hw.logicalcpu, hw.logicalcpu_max, hw.physicalcpu, or
             hw.physicalcpu_max be used instead.